dsm/common.go

```diff
@@ -263,4 +264,91 @@ func undoTFstate(param_type string, d *schema.ResourceData) diag.Diagnostics {
 		  Detail:   fmt.Sprintf("[E]: API: PATCH crypto/v1/keys: %s cannot change on update. Please retain it to old value: %s -> %s", param_type, old_val, new_val),
 	})
 	return diags
-}
\ No newline at end of file
+}
+
+// expiry_date create and update
```

Comment:
This comment should be improved to describe what the function does or simply omit it entirely

Reply:
Enhanced.